From: Kentaro Hayashi <kenhys@gmail.com>
Date: Sat, 18 May 2024 17:05:16 +0900
Subject: Fix missing abseil libraries

It fixes the following error:

  /usr/bin/ld: obj/base/base_core.file_util.o: undefined reference to
  symbol '_ZN4absl7debian5lsERSoNS0_11string_viewE'
  /usr/bin/ld:
  /lib/x86_64-linux-gnu/libabsl_string_view.so.20230802: error adding
  symbols: DSO missing from command line

  /usr/bin/ld: obj/unix/emacs/mozc_emacs_helper_lib.client_pool.o:
  undefined reference to symbol
  '_ZN4absl7debian516raw_log_internal21internal_log_functionB5cxx11E'
  /usr/bin/ld:
  /lib/x86_64-linux-gnu/libabsl_raw_logging_internal.so.20230802: error
  adding symbols: DSO missing from command line collect2: error: ld
  returned 1 exit status

Signed-off-by: Kentaro Hayashi <kenhys@gmail.com>
---
 src/base/absl.gyp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/base/absl.gyp b/src/base/absl.gyp
index 4bacb59..6105498 100644
--- a/src/base/absl.gyp
+++ b/src/base/absl.gyp
@@ -47,7 +47,7 @@
         ['use_libabseil==1', {
           'link_settings': {
             'libraries': [
-              '-latomic -labsl_base -labsl_int128 -labsl_base -labsl_hash -labsl_city -labsl_flags_reflection -labsl_raw_hash_set -labsl_str_format_internal -labsl_throw_delegate -labsl_time_zone -labsl_hashtablez_sampler -labsl_synchronization -labsl_time -labsl_strings_internal -labsl_strings -labsl_spinlock_wait -labsl_status -labsl_statusor -labsl_flags_internal -labsl_flags_usage_internal -labsl_flags_marshalling -labsl_flags_parse',
+              '-latomic -labsl_base -labsl_int128 -labsl_base -labsl_hash -labsl_city -labsl_flags_reflection -labsl_raw_hash_set -labsl_str_format_internal -labsl_throw_delegate -labsl_time_zone -labsl_hashtablez_sampler -labsl_synchronization -labsl_time -labsl_strings_internal -labsl_strings -labsl_spinlock_wait -labsl_status -labsl_statusor -labsl_flags_internal -labsl_flags_usage_internal -labsl_flags_marshalling -labsl_flags_parse -labsl_string_view -labsl_raw_logging_internal',
             ],
           },
         },],
